aws/logs_monitoring/lambda_function.py

```diff
@@ -57,6 +62,8 @@
     "\d+_CloudTrail_\w{2}-\w{4,9}-\d_\d{8}T\d{4}Z.+.json.gz$", re.I
 )
 ip_regex = re.compile("\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}", re.I)
+email_regex = re.compile("(\"|\')[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+(\"|\')", re.I)
```

Comment:
Did some test and it seems that you would need to remove `(\"|\')` at the begining and at the end of the regex to have this working.